### PR TITLE
Add uninstall information to foreman_templates

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by ares** [comment]
@GregSutcliffe mind to take a look?

File: plugins/foreman_templates/4.0/index.md [hunk 1]
```diff
@@ -12,13 +12,24 @@ You can use the native git support for simple synchronization with remote git re
 
 # 2. Installation
 
+# 2.1 Installation
```
Comment (GregSutcliffe) [inline plugins/foreman_templates/4.0/index.md]: You have "Installation" twice here. I'd drop the 2.1 heading and allow Uninstallation to become 2.1, I think that looks better.

**Comment by ares** [comment]
Thanks, updated. Could you please take another look?

**Comment by GregSutcliffe** [comment]
Done, thanks @ares !

**Pull request merged**